### PR TITLE
Work cycles

### PR DESCRIPTION
**Pull request by hackartisan**

### CODE REVIEW

File: work_cycles.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Our team works in 2-week cycles, aka "sprints". We usually split into 2 sub-teams in each
+cycle, with each sub-team. This means that we do not work on every application
+at all times.
+
+We track all our work in github issues and zenhub boards. Each project has a github repository. In most cases the issues from that repository are organized into columns on the corresponding zenhub board. These resources are listed on the [List of Applications](https://github.com/pulibrary/app_team_handbook/blob/master/applications.md)
+
+At the beginning of each cycle we have a planning meeting to triage incoming
+issues and decide which issues we will be working on during that cycle. Issues
+we plan to work during that cycle are placed in the "ready" column of the zenhub
```
Comment (jrgriffiniii) [inline work_cycles.md:12]: Is the "icebox" column worth referencing in this section, or is this too specific to just Figgy? I would contend that the "inbox" to "ready" workflow is best left in the documentation for the Runner role.

**Review by jrgriffiniii — APPROVED** [review]

File: work_cycles.md (hunk 1 above)
Comment (hackartisan) [reply work_cycles.md:12]: I was thinking that in the context of work cycles getting into the icebox would be a bit deep. Maybe we should have a ticket for explaining our board setup with more details? Interested in more thoughts here.

File: work_cycles.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# DRDS Work Cycles and Practices
+
+## Work Cycles
+
+Our team works in 2-week cycles, aka "sprints". We usually split into 2 sub-teams in each
+cycle, with each sub-team. This means that we do not work on every application
```
Comment (hackartisan) [inline work_cycles.md]: should read "with each sub-team working on a different application or project"

**Review by tpendragon — CHANGES_REQUESTED** [review]
Looks good, I'd just like to see the "each sub-team working on a different application or project" get in, then we should merge.

**Review by tpendragon — APPROVED** [review]

File: work_cycles.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# DRDS Work Cycles and Practices
+
+## Work Cycles
+
+Our team works in 2-week cycles, aka "sprints". We usually split into 2 sub-teams in each
+cycle, with each sub-team working on a different application or project. This means that we do not work on every application
```
Comment (kelea99) [inline work_cycles.md]: Maybe reword this? I'm trying to keep your voice, promise. "This means that we work on a prioritized subset of applications for each sprint." or something like that?

File: work_cycles.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+At the beginning of each cycle we have a planning meeting to triage incoming
+issues and decide which issues we will be working on during that cycle. Issues
+we plan to work during that cycle are placed in the "ready" column of the zenhub
+board. We order them roughly by priority. Over the course of the cycle individual developers will select issues, move them to the "in progress" column, and mark them assigned. When issues are complete they'll be closed.
+
+## Communication
+
+Our team communicates primarily over slack. This allows us to quickly and easily
+reach out to one another and to product owners and other stakeholders. While writing code often happens individually, software development is a highly collaborative activity. Knowledge about our applications, systems, interfaces, users, and use cases are distributed throughout our team and other teams.
```
Comment (kelea99) [inline work_cycles.md]: "and other teams". give examples? such as...? just a suggestion.

File: work_cycles.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+We use github as outlined above to record use cases, progress on work, and decisions that impact the work on a particular issue.
+
+We also frequently communicate on zoom. We hold all meetings this way and our
+team culture is to attend meetings with video activated when possible. We also
+use zoom throughout the day to demonstrate software functionality or bugs, or
+when a slack conversation becomes unwieldy.
+
+Finally, we use Google drive record notes for meetings or produce design
```
Comment (kelea99) [inline work_cycles.md]: ....google drive _to_  record notes for meetings...

File: work_cycles.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+documents.
+
+## Technical Leads
+
+Each of our projects has an assigned technical lead. This person will work most
```
Comment (kelea99) [inline work_cycles.md:32]: re: tech leads-- perhaps cover this important point more clearly for the PO documentation- and (re-)affirmation what expectations are in regards to the lead and the PO (the communication info, for example, below), and refer back to who the tech leads are for each app in that same documentation? It was really helpful for me to see an actual list of the tech leads.  does this Make sense?

**Review by kelea99 — COMMENTED** [review]
Looks good! A few comments below. take or leave as you see fit. Thank you!

File: work_cycles.md (hunk 6 above)
Comment (hackartisan) [reply work_cycles.md:32]: The link to applications is in that other one. I've just add description to say that it includes the list of tech leads.

I'm not entirely sure I understand this comment. Are you asking for more info on the PO page or on the TL page? Or are you asking for something on this page?

**Comment by hackartisan** [comment]
@kelea99 Otherwise I've made edits in response to your review. Thank you!

File: work_cycles.md (hunk 6 above)
Comment (kelea99) [reply work_cycles.md:32]: Doh. Nope. you dont understand the comment, because it makes no sense! Sorry.  I Was reading one doc and didn't remember the link references from the other I read the evening before. Disregard.
